misc/gui_linux.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+
+global=$(zenity --forms --add-entry="Round Name" --add-entry="Round Title" --add-entry="Main User" --add-entry="OS" --add-entry="Remote" --add-entry="Local" --add-entry="EndDate" --add-entry="NoDestroy" --separator=$'\034' --title=AeacusSE)
+
+IFS=$'\034' read -r name title user os remote local end destroy <<< "$global"
+
+next=""
+vulns=""
+
+while [[ "$next" != "None" ]]
+do
+
+next=$(zenity --list --title="Add vulnerabilities" --text "Which category do you want to add?" --radiolist --column "Pick" --column "Vuln Category" FALSE "Command" FALSE "CommandNot" FALSE "FileExists" FALSE "FileExistsNot" FALSE "FileContains" FALSE "FileContainsNot" FALSE "FileContainsRegex" FALSE "FileContainsRegexNot" FALSE "DirContainsRegex" FALSE "DirContainsRegexNot" FALSE "FileEquals" FALSE "FileEqualsNot" FALSE "PackageInstalled" FALSE "PackageInstalledNot" FALSE "ServiceUp" FALSE "ServiceUpNot" FALSE "UserExists" FALSE "UserExistsNot" FALSE "FirewallUp" FALSE "UserInGroup" FALSE "UserInGroupNot" FALSE "GuestDisabledLDM" FALSE "None" --height=700 --width=600 --title=AeacusSE)
```

Comment:
Can we get the appropriate tabbing in here?

Reply:
Can't do that because zenity doesn't support tabs.

Reply:
Oh ok got it